apps/modernization-ui/src/apps/page-builder/pages/EditPage/ManageTabs/ManageTabs.scss

```diff
@@ -0,0 +1,35 @@
+@import 'settings.scss';
+
+.manage-tabs {
+    .manage-tabs__tile {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        color: $color-black;
+        padding: 1rem 1.5rem;
+    }
+    .manage-tabs__label {
+        display: flex;
+        gap: 1rem;
+    }
+    .manage-tabs__buttons {
+        color: $color-primary;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        gap: 0.4rem;
+        svg {
+            cursor: pointer;
+            &:hover {
+                color: $color-primary-light;
+            }
+            &:nth-of-type(2) {
+                &:hover {
+                    color: $color-alert-error;
+                }
+            }
+        }
+    }
+}
+
+
```

Comment:
Does this still match the appropriate styling? It prevents the full import of the entire `settings.scss`

```scss
@use 'settings.scss';

.manage-tabs {
    .manage-tabs__tile {
        display: flex;
        flex-direction: row;
        justify-content: space-between;
        color: settings.$color-black;
        padding: 1rem 1.5rem;
    }
    .manage-tabs__label {
        display: flex;
        gap: 1rem;
    }
    .manage-tabs__buttons {
        color: settings.$color-primary;
        display: flex;
        justify-content: center;
        align-items: center;
        gap: 0.4rem;
        svg {
            cursor: pointer;
            &:hover {
                color: settings.$color-primary-light;
            }
            &:nth-of-type(2) {
                &:hover {
                    color: settings.$color-alert-error;
                }
            }
        }
    }
}

```